CyclomaticComplexityAssessorRunner

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env php
 <?php
-require_once 'vendor/autoload.php';
+require_once __DIR__ . '/vendor/autoload.php';
 
 use Churn\Assessors\CyclomaticComplexity\CyclomaticComplexityAssessor;
 
 $file = $argv[1];
 $assessor = new CyclomaticComplexityAssessor();
-echo $assessor->assess($file);
\ No newline at end of file
+echo $assessor->assess($file);
```

Comment:
I don't think you'll need to make any changes in this file.

Reply:
- The `__DIR__` change is necessary to include the correct autoload file.
- Every editor I have will insert the newline at EOF. I did not actually notice it before.